server/cdk/assets/app.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=It's the backend
+Requires=network.target
+After=network.target
```

Comment:
Is both After and Requires both needed?

Reply:
https://stackoverflow.com/questions/40593242/systemd-using-both-after-and-requires
Apparently both is good to have